tests/codegen/mpi_axpy.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+import argparse
+import dace
+import numpy as np
+import scipy as sp
+from mpi4py import MPI
+from dace.transformation.dataflow import MPITransformMap
+
+N = dace.symbol('N')
+
+
+@dace.program(dace.float64, dace.float64[N], dace.float64[N])
+def axpy(A, X, Y):
```

Comment:
Is this related to this PR?

Reply:
Yes, see above comment about MPI tests being redone.